src/components/layout/PanelChange/index.tsx

```diff
@@ -318,7 +366,7 @@ export const PanelChange: FC<IProps> = ({
 								<div className={styles.stream}>
 									<span>
 										<span className={styles.number}>
-											{`$${personalFlow && getFlowUSDValue(personalFlow)} ${t('per month')}`}
+											<>{`$${personalFlow && getFlowUSDValue(personalFlow)} ${t('per month')}`}</>
```

Comment:
Why are you using fragments here? Is it required? Just adds extra dom elements and slows things down